src/parser/expression.js

```diff
@@ -385,6 +385,16 @@ pp.parseExprAtom = function (refShorthandDefaultPos) {
       }
       return this.finishNode(node, "Super");
 
+    case tt._import:
+      if (!this.hasPlugin("importFunctions")) this.unexpected();
```

Comment:
Doesn't have to be done in this PR since it's a separate issue but we should make a better error message here (when we have a `this.hasPlugin(`)